src/scalar.rs

```diff
@@ -905,15 +906,15 @@ fn test_inv() {
 fn test_debug() {
     assert_eq!(
         format!("{:?}", Scalar::zero()),
-        "0x0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Notice that you can keep the `0x` form just adding the alternate symbol to the format, so where `{:?}` produces "0000…" (etc); `{:#?}` produces `0x0000`.

Reply:
Both tested and working fine